setup.py

```diff
@@ -57,7 +57,7 @@
 # Development dependencies (*not* included in 'all')
 EXTRAS['dev'] = [
     # Please keep alphabetized
-    'baselines @ https://api.github.com/repos/openai/baselines/tarball/f2729693253c0ef4d4086231d36e0a4307ec1cb3',  # noqa: E501
+    'baselines @ https://api.github.com/repos/openai/baselines/tarball/ea25b9e8b234e6ee1bca43083f8f3cf974143998',  # noqa: E501
```

Comment:
don't forget to use the updated style, which uses `GARAGE_GH_TOKEN` so we don't run afoul of GitHub API limits.